dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEventAnalyticsManager.java

```diff
@@ -551,6 +551,20 @@ else if ( params.isOrganisationUnitMode( OrganisationUnitSelectionMode.CHILDREN
                 + params.getEventStatus().stream().map( e -> "'" + e.name() + "'" ).collect( joining( "," ) ) + ") ";
         }
 
+        if ( params.hasCreatedBy() )
+        {
+            sql += hlp.whereAnd() + " createdby in ("
+                + params.getCreatedBy().stream().map( username -> "'" + username + "'" ).collect( joining( "," ) )
```

Comment:
I think it's better to use
```
org.hisp.dhis.analytics.util.AnalyticsSqlUtils#encode
```
to avoid SQL injection

Reply:
Good point. Done.